pkg/rancher-desktop/backend/lima.ts

```diff
@@ -789,11 +789,20 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
   /**
    * Run `limactl` with the given arguments.
    */
-  async lima(this: Readonly<this>, ...args: string[]): Promise<void> {
+  async lima(this: Readonly<this>, env: NodeJS.ProcessEnv, ...args: string[]): Promise<void>;
+  async lima(this: Readonly<this>, ...args: string[]): Promise<void>;
+  async lima(this: Readonly<this>, envOrArg: NodeJS.ProcessEnv | string, ...args: string[]): Promise<void> {
+    const env = LimaBackend.limaEnv;
+
+    if (typeof envOrArg === 'string') {
+      args = [envOrArg].concat(args);
+    } else {
+      Object.assign(env, envOrArg);
+    }
```

Comment:
This seems to complex. There is no reason not to set the variable in `LimaBackend.limaEnv` instead:

```ts
if (this.cfg.experimental.sshPortForwarder) {
  env.LIMA_SSH_PORT_FORWARDER = 'true'
}
```

and then the rest of the changes here are not necessary.

Reply:
The hard part was that it was a static method, so there is no `this.cfg`. It was actually my first attempt (making it a method and all that), but I realized that it meant we always set that on every command (not just `limactl start`), which seemed wrong as well.

Reply:
> I realized that it meant we always set that on every command (not just `limactl start`), which seemed wrong as well.

It doesn't matter. It would be the same thing if you set it in your `~/.bash_profile`. The variable is not used for anything else.

Reply:
It matters in that it makes it easier for me to remember that the environment variable is only relevant on `limactl start`; this just keeps the thinking clearer, it doesn't matter if the actual command wants it.